fedimint-server-ui/src/dashboard.rs

```diff
@@ -183,33 +156,47 @@ async fn dashboard_view(
                 (latency::render(consensus_ord_latency, &p2p_connection_status))
             }
         }
+
+        (lightning_content)
+        (wallet_content)
+        (meta_content)
+
+        // Every 15s fetch updates to the page
+        div hx-get="/dashboard/update" hx-trigger="every 15s" hx-swap="none" { }
     };
 
-    // Conditionally add Lightning V2 UI if the module is available
-    if let Some(lightning) = state.api.get_module::<fedimint_lnv2_server::Lightning>() {
-        content = html! {
-            (content)
-            (lnv2::render(lightning).await)
-        };
-    }
+    Html(dashboard_layout(content).into_string()).into_response()
+}
 
-    // Conditionally add Wallet UI if the module is available
-    if let Some(wallet_module) = state.api.get_module::<fedimint_wallet_server::Wallet>() {
-        content = html! {
-            (content)
-            (wallet::render(wallet_module).await)
-        };
+/// Periodic updated to the dashboard
+///
+/// We don't just replace the whole page, not to interfere with elements that
+/// might not like it.
+async fn dashboard_update(
+    State(state): State<AuthState<DynDashboardApi>>,
+    jar: CookieJar,
+) -> impl IntoResponse {
+    if !check_auth(&state.auth_cookie_name, &state.auth_cookie_value, &jar).await {
```

Comment:
Note: we should move auth out so we don't need to remember about doing it on every endpoint.